src/main/java/uk/gov/cshr/civilservant/controller/QuizController.java

```diff
@@ -38,14 +47,58 @@
                 .orElseGet(() -> ResponseEntity.noContent().build());
     }
 
-    @PostMapping
-    @Transactional
+   @PostMapping("/create")
```

Comment:
We shouldn't need the `("/create")` path here. Following good rest naming conventions, a post to ("/quizzes") should create our new quiz.

Reply:
Same with the endpoints below